src/Components/Blueprints/BuildImageButtonEmpytyState.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import { Button, ButtonProps } from '@patternfly/react-core';
+
+import { selectSelectedBlueprintId } from '../../store/BlueprintSlice';
+import { useAppSelector } from '../../store/hooks';
+import { useComposeBlueprintMutation } from '../../store/imageBuilderApi';
+
+type BuildImagesButtonEmptyStatePropTypes = {
+    variant?: ButtonProps['variant'];
+    children?: React.ReactNode;
+};
+
+export const BuildImagesButtonEmptyState = ({
```

Comment:
IMO this component could live in the same file (BuildImagesButton), there is no strong need to have it separate.